src/arguments.js

```diff
@@ -179,33 +180,34 @@ const removeExtraSpaces = (string) => {
 };
 
 /**
-* Remove extra coma from the end of line
-* @param {string} string - any string
-* @return {string} - result string
-*/
-const removeComaAtTheEnd = (string) => {
-    return removeCharAtTheEnd(string, ',');
-};
-
-/**
-* Remove extra dot from the end of line
-* @param {string} string - any string
-* @return {string} - result string
-*/
-const removeDotAtTheEnd = (string) => {
-    return removeCharAtTheEnd(string, '.');
+ * Delete chars at the end of string
+ * @param {string} string - source string
+ * @param {array} array - char for remove
+ * @return {*} - string without chars
+ */
+const removeCharAtTheEnd = (string, array) => {
+    let result = string;
+    array.map((char) => {
+        result = result.charAt(result.length - 1) === char ?
+            result.slice(0, result.length - 1) : result;
+    });
+    return result;
 };
 
-
 /**
-* Delete char at the end of string
+* Delete chars at the beggining and at the end of string
 * @param {string} string - source string
-* @param {char} char - char for remove
-* @return {*} - string without dot at the end
+* @param {array} array - char for remove
+* @return {*} - string without chars
 */
-const removeCharAtTheEnd = (string, char) => {
-    return string.charAt(string.length - 1) === char ?
-        string.slice(0, string.length - 1) : string;
+const removeCharAtSides = (string, array) => {
```

Comment:
I think we need to change this function to `removeCharAtTheStart` or combine it with `removeCharAtTheEnd` (then remove last).  Support both function there is no point